.gitignore

```
@@ -129,3 +131,4 @@ _al_test_project_name*
 package-lock.json
 #report logs
 /debug_log.txt
+/log.txt
```

Sorry, at this point log stuff needs to be removed. I didn't catch anything elsewhere, but this can be cleaned up.

```suggestion
/debug_log.txt
```

---

This was intentional, I had rebased and accudentally commited my log.txt to the repo. I think that others will likely make the same mistake, and IMO it's doesn't need to be removed from gitignore anyway.

---

I think there are only three of us with the problem and we've all dealt with it, but it won't do any harm. Sound good then 👍